pytorch_lightning/trainer/trainer.py

```diff
@@ -257,6 +259,15 @@ def __init__(
         """
         super().__init__()
 
+        # deprecation warnings
+        if row_log_interval is not None:
```

Comment:
move this to class with deprecated API, so the Trainer is clean...

Reply:
Deprecation warnings for arguments remain in trainer class, see https://github.com/PyTorchLightning/pytorch-lightning/blob/92d6abcbb9e73645fff0bba2914f7a7e0e748a91/pytorch_lightning/trainer/trainer.py#L575-L578